message-templates/mjml/dataset-revision.mjml

```diff
@@ -78,7 +78,26 @@
               <center>
             <![endif]-->
                 <a href="https://${discoverHost}/datasets/${discoverDatasetId}"
-          style="background-color:#5039F7;border-radius:3px;color:#ffffff;display:inline-block;font-family:sans-serif;font-size:14px;font-weight:500;line-height:40px;text-align:center;text-decoration:none;width:200px;-webkit-text-size-adjust:none;">View now</a>
+          style="background-color:#5039F7;border-radius:3px;color:#ffffff;display:inline-block;font-family:sans-serif;font-size:14px;font-weight:500;line-height:40px;text-align:center;text-decoration:none;width:200px;-webkit-text-size-adjust:none;">View Dataset on Discover</a>
+            <!--[if mso]>
+              </center>
+            </v:roundrect>
+          <![endif]-->
+        </mj-text>
+      </mj-column>
+    </mj-section>
+
+    <mj-section mj-class="copy-section">
+      <mj-column padding="16px 0">
+        <mj-text height="40px">
+          <!--[if mso]>
+            <v:roundrect xmlns:v="urn:schemas-microsoft-com:vml" xmlns:w="urn:schemas-microsoft-com:office:word" href="https://${discoverHost}/datasets/${discoverDatasetId}" style="height:40px;v-text-anchor:middle;width:200px;" arcsize="8%" stroke="f" fillcolor="#5039F7">
+              <w:anchorlock/>
+              <center>
+            <![endif]
+                MIGHT NEED TO CHANGE THE NODE IDS AND BUTTON VISUALS-->
+                <a href="https://app.pennsieve.io/${organizationNodeId}/datasets/${datasetNodeId}/activity/logs"
```

Comment:
We should use a parameter to specify the App host here. The same is done in 'added-to-organization.mjml' (look for where ${host} is used to form a URL, and the code using it. I can help track down where in the Scala code we do this -- just let me know.